R/civis_ml_utils.R

```diff
@@ -20,8 +20,8 @@ CIVIS_ML_CLASSIFIERS <- c("sparse_logistic",
                           "stacking_classifier")
 
 CIVIS_ML_TEMPLATE_IDS <- data.frame(
-  id = c(9112, 9113, 9968, 9969),
-  version = c(1.1, 1.1, 2.0, 2.0),
+  id = c(9112, 9113, 9968, 9969, 10582, 10583),
+  version = c(1.1, 1.1, 2.0, 2.0, 2.1, 2.1),
   name = c("train", "predict", "train", "predict"),
```

Comment:
It looks like the `name` column needs and extra `c('train', 'predict')`. This might actually build correctly, because R will loop through the vector again to fill in the column, but it's kind of a hacky behavior on R's part, and I don't think we should rely on it.

Reply:
Oops, I was looking at an old commit. I see now that you updated this.